rel/i18n/emqx_mgmt_api_key_schema.hocon

```diff
@@ -9,8 +9,9 @@ api_key.label:
 bootstrap_file.desc:
 """The bootstrap file provides API keys for EMQX.
 EMQX will load these keys on startup to authorize API requests.
-It contains key-value pairs in the format:`api_key:api_secret`.
-Each line specifies an API key and its associated secret."""
+It contains key-value pairs in the format:`api_key:api_secret:role`.
+Each line specifies an API key and its associated secret.
+`role` is optional, its default value is `administrator`."""
```

Comment:
```suggestion
It contains colon-separated values in the format:`api_key:api_secret:role`.
Each line specifies an API key and its associated secret, and the role of the this key.
The 'role' part is introduced since 5.4. It should be the pre-defined access scope group name,
for example, `administrator` or `viewer`. 
If 'role' is missing, it defaults to `administrator`."""
```